### PR TITLE
add owners file support

### PR DESCRIPTION
**Pull request by edisonxiang**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
* /lgtm && /lgtm cancel
* /approve && /approve cancel

**Which issue(s) this PR fixes**: #6

### CODE REVIEW

File: handlers/approve/approve.go [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+		}
+		// cancel approve
+		if RegCancelApprove.MatchString(comment) {
+			return Cancel(client, r, event)
+		}
+	}
+	return nil
+}
+
+// Add approve
```
Comment (m1093782566) [inline handlers/approve/approve.go]: Add approve label

File: handlers/approve/approve.go [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+			glog.Infof("Add label successfully: %v", listOfAddLabels)
+		}
+	} else {
+		glog.Infof("No label to add: %v", LabelNameApproved)
+	}
+
+	return nil
+}
+
+// Cancel approve
```
Comment (m1093782566) [inline handlers/approve/approve.go]: Cancel removes approve label

File: handlers/approve/approve.go (hunk 1 above)
Comment (edisonxiang) [reply handlers/approve/approve.go]: Thanks, will update it later

File: handlers/approve/approve.go (hunk 2 above)
Comment (edisonxiang) [reply handlers/approve/approve.go]: Thanks, will update it later

**Comment by sids-b** [comment]
@edisonxiang , have you tested this feature on a sample repository ?

**Comment by edisonxiang** [comment]
@sids-b yep. I tested it in my repository:)
https://github.com/edisonxiang/ci-bot-test

**Comment by sids-b** [comment]
> @sids-b yep. I tested it in my repository:)
> https://github.com/edisonxiang/ci-bot-test

Great!! Maybe we can merge this now and keep adding/improving features.

**Review by sids-b — APPROVED** [review]
LGTM

**Comment by edisonxiang** [comment]
@sids-b Thanks

File: handlers/approve/approve.go (hunk 1 above)
Comment (edisonxiang) [reply handlers/approve/approve.go]: Done

File: handlers/approve/approve.go (hunk 2 above)
Comment (edisonxiang) [reply handlers/approve/approve.go]: Done

**Pull request merged**